### PR TITLE
docs updates 1/n

### PR DESCRIPTION
**Pull request by williamFalcon**
make docs great again

### CODE REVIEW

File: requirements/pytorch/docs.txt [hunk 1]
```diff
@@ -1,7 +1,6 @@
 -r ../docs.txt
 
 ipython[notebook]
-setuptools<58.0  # workaround for `error in ipython setup command: use_2to3 is invalid.`
-pt-lightning-sphinx-theme @ https://github.com/Lightning-AI/lightning_sphinx_theme/archive/master.zip
+pt-lightning-sphinx-theme @ https://github.com/Lightning-AI/lai_sphinx_theme/archive/master.zip
```
Comment (carmocca) [inline requirements/pytorch/docs.txt:4]: Probably an unintended change

```suggestion
setuptools<58.0 # workaround for `error in ipython setup command: use_2to3 is invalid.`
pt-lightning-sphinx-theme @ https://github.com/Lightning-AI/lai_sphinx_theme/archive/master.zip
```

File: requirements/docs.txt [hunk 2]
```diff
@@ -5,9 +5,13 @@ pandoc>=1.0, <=2.2
...
+sphinx-autobuild
 sphinx-autodoc-typehints>=1.16
 sphinx-paramlinks>=0.5.1, <=0.5.4
 sphinx-togglebutton>=0.2, <=0.3.2
 sphinx-copybutton>=0.3, <=0.5.0
 sphinx-multiproject
+sphinx-toolbox==3.2.0
+sphinx-rtd-dark-mode
 jinja2>=3.0.0,<3.1.0
+pt-lightning-sphinx-theme @ https://github.com/Lightning-AI/lai_sphinx_theme/archive/master.zip
```
Comment (carmocca) [inline requirements/docs.txt:17]: I would suggest to add this here or in `requirements/*/docs.txt`, but not both. I don't know which place is better. Maybe @Borda knows.

File: requirements/base.txt [hunk 3]
```diff
@@ -0,0 +1,17 @@
+lightning-cloud==0.5.7
```
Comment (carmocca) [inline requirements/base.txt:1]: I don't think this file was added intentionally.

File: node_modules/.yarn-integrity [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (carmocca) [inline node_modules/.yarn-integrity:1]: I don't think this file was added intentionally. Add it to gitignore?

File: docs/source-pytorch/starter/installation.rst [hunk 5]
```diff
@@ -59,3 +74,31 @@ Install future patch releases from the source. Note that the patch release conta
 .. code-block:: bash
 
     pip install https://github.com/Lightning-AI/lightning/archive/refs/heads/release/stable.zip -U
+
+----
+
+*******************************
+Optimized for model development
+*******************************
+If you are deploying models built with Lightning in production and require few dependencies, try using the optimized `lightning[pytorch]` package:
```
Comment (carmocca) [inline docs/source-pytorch/starter/installation.rst:83]: This tests suggests something different to the snippet right below.

File: docs/source-pytorch/conf.py [hunk 6]
```diff
@@ -178,9 +185,9 @@ def _transform_changelog(path_in: str, path_out: str) -> None:
 # http://www.sphinx-doc.org/en/master/usage/theming.html#builtin-themes
 # html_theme = 'bizstyle'
 # https://sphinx-themes.org
-html_theme = "pt_lightning_sphinx_theme"
-html_theme_path = [pt_lightning_sphinx_theme.get_html_theme_path()]
-# html_theme_path = ["/Users/williamfalcon/Developer/opensource/lightning_sphinx_theme"]
+html_theme = "lai_sphinx_theme"
+html_theme_path = ["/Users/williamfalcon/Developer/sphinx-theme"]
```
Comment (carmocca) [inline docs/source-pytorch/conf.py:189]: Might need to be reverted before merging

File: docs/source-lit/conf.py [hunk 7]
```diff
@@ -155,8 +162,8 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = "pt_lightning_sphinx_theme"
-html_theme_path = [pt_lightning_sphinx_theme.get_html_theme_path()]
+html_theme = "lai_sphinx_theme"
+html_theme_path = ["/Users/williamfalcon/Developer/sphinx-theme"]
```
Comment (carmocca) [inline docs/source-lit/conf.py:166]: Same here

File: docs/source-app/conf.py [hunk 8]
```diff
@@ -149,8 +151,8 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = "pt_lightning_sphinx_theme"
-html_theme_path = [pt_lightning_sphinx_theme.get_html_theme_path()]
+html_theme = "lai_sphinx_theme"
+html_theme_path = ["/Users/williamfalcon/Developer/lightning_theme"]
```
Comment (carmocca) [inline docs/source-app/conf.py:155]: Same reminder

File: requirements/docs.txt (hunk 2 above)
Comment (Borda) [reply requirements/docs.txt]: it is already there

**Comment by williamFalcon** [comment]
this is a draft...

File: docs/README.md [hunk 9]
```diff
@@ -72,5 +72,5 @@ export PL_FAST_DOCS_DEV=1
 
 ## docs CSS/theme
 
-To change the CSS theme of the docs, go [here](https://github.com/Lightning-AI/lightning_sphinx_theme).
+To change the CSS theme of the docs, go [here](https://github.com/Lightning-AI/lai_sphinx_theme).
```
Comment (Borda) [inline docs/README.md:75]:
```suggestion
To change the CSS theme of the docs, go [here](https://github.com/Lightning-AI/sphinx-theme).
```

File: docs/source-app/levels/basic/hello_components/deploy_model.py [hunk 10]
```diff
@@ -0,0 +1,15 @@
+# A hello world component
```
Comment (Borda) [inline docs/source-app/levels/basic/hello_components/deploy_model.py:1]: I think we had some problem with having py files/examples in docs so we moved them to examples and with docs build time copied them...

File: docs/source-app/levels/expert/index.rst [hunk 11]
```diff
@@ -0,0 +1,93 @@
+.. _expert_level:
+
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+
+#############
+Expert skills
+#############
+??
```
Comment (Borda) [inline docs/source-app/levels/expert/index.rst:10]:
```suggestion
```

File: docs/source-app/levels/basic/run_jupyter_notebook_on_the_cloud.rst [hunk 12]
```diff
@@ -0,0 +1,19 @@
+#############################################
+Example: Develop a Jupyter Notebook component
+#############################################
+**Audience:** TODO:
```
Comment (Borda) [inline docs/source-app/levels/basic/run_jupyter_notebook_on_the_cloud.rst:4]: I see several TODO, are we addressing them in this PR or another/following?

**Review by Borda — APPROVED** [review]
looks fine, let me also check the CI
we shall ignore building lit docs and fix it in following PR

**Pull request merged**